### PR TITLE
upgrade wavesuffer to 7.3.4

### PR DESCRIPTION
**Pull request by xquanluu**

### CODE REVIEW

**Comment by xquanluu** [comment]
Screenshot 2023-10-12 at 12 41 14

File: src/containers/internal/views/recent-calls/player.tsx [hunk 1]
```diff
@@ -46,15 +46,19 @@ export const Player = ({ call }: PlayerProps) => {
 
   const wavesurferId = `wavesurfer--${call_sid}`;
   const wavesurferTimelineId = `timeline-${wavesurferId}`;
-  const waveSufferRef = useRef<WaveSurfer | null>(null);
+  const waveSuferRef = useRef<WaveSurfer | null>(null);
```
Comment (davehorton) [inline src/containers/internal/views/recent-calls/player.tsx:49]: can you fix the typos?  "waveSufer" should be "waveSurfer"

**Review by davehorton — CHANGES_REQUESTED** [review]

**Comment by davehorton** [comment]
There is no change in functionality here, is that right?  Just upgrading to the most recent version of wavesurfer?

**Comment by xquanluu** [comment]
yes it's correct

The timeline ruler is more detail in 0.2 second

**Comment by davehorton** [comment]
ok great.  After fixing the comment above can you deploy to jambonz.me for testing?

**Comment by xquanluu** [comment]
fixed typo issue

**Review by davehorton — APPROVED** [review]

**Pull request merged**